kalite/playlist/models.py

```diff
@@ -130,7 +131,7 @@ def all(cls, limit_to_shown=True):
 
             playlists.append(playlist)
 
-        return playlists
+        return playlists if "Nalanda" in settings.CONFIG_PACKAGE else []
```

Comment:
It might be good to do this if statement earlier in the function, to avoid doing the work in the first place.